platform/integrations/slack_integration/setup.mdx

```diff
@@ -0,0 +1,42 @@
+---
+id: setup
+title: Setup
+sidebar_label: Setup
+description: Learn how to install and setup Tolgee Slack app. First, you need to install the app from the Slack App Directory by clicking on the Add to Slack button.
+---
+
+import { FigmaScreenshotWrapper } from '../../shared/_FigmaScreenshotWrapper';
+
+## Installing the Tolgee Integration for Slack
+
+### Prerequisites
+
+Before you begin, ensure you have:
+
+- A Tolgee account with the necessary permissions to manage integrations.
+- A Slack workspace where you have permission to add apps.
+
+### Step-by-Step Guide
+
+#### Step 1: Install the Tolgee App in Slack
+
+Install Slack application from [Slack App Directory](https://slack.com/apps/dumbLink) by clicking on the **Add to Slack** button. Or you can find app right in Slack by
```

Comment:
This is not the flow we want the user to follow. We want to install the app from Tolgee platform and it will install it to the workspace. This should contain a screenshot.